app/javascript/dashboard/routes/dashboard/settings/auditlogs/Index.vue

```diff
@@ -1,137 +1,126 @@
+    />
+
+    <div class="mt-6 flex-1 text-slate-700 dark:text-slate-300">
+      <woot-loading-state
+        v-if="uiFlags.fetchingList"
+        :message="$t('AUDIT_LOGS.LOADING')"
+      />
+      <p
+        v-else-if="!records.length"
+        class="flex flex-col items-center justify-center h-full text-base p-8"
+      >
+        {{ $t('AUDIT_LOGS.LIST.404') }}
+      </p>
 
-        <table
-          v-if="!uiFlags.fetchingList && records.length"
-          class="w-full woot-table"
+      <table
+        v-else
+        class="min-w-full divide-y divide-slate-75 dark:divide-slate-700"
+      >
+        <thead>
+          <th
+            v-for="thHeader in $t('AUDIT_LOGS.LIST.TABLE_HEADER')"
+            :key="thHeader"
+            class="py-4 pr-4 text-left font-semibold text-slate-700 dark:text-slate-300"
+          >
+            {{ thHeader }}
+          </th>
+        </thead>
+        <tbody
+          class="divide-y divide-slate-50 dark:divide-slate-800 text-slate-700 dark:text-slate-300"
         >
-          <colgroup>
-            <col class="w-3/5" />
-            <col />
-            <col />
-          </colgroup>
-          <thead>
-            <!-- Header -->
-            <th
-              v-for="thHeader in $t('AUDIT_LOGS.LIST.TABLE_HEADER')"
-              :key="thHeader"
-            >
-              {{ thHeader }}
-            </th>
-          </thead>
-          <tbody>
-            <tr v-for="auditLogItem in records" :key="auditLogItem.id">
-              <td class="break-all whitespace-nowrap">
-                {{ generateLogText(auditLogItem) }}
-              </td>
-              <td class="break-all whitespace-nowrap">
-                {{
-                  messageTimestamp(
-                    auditLogItem.created_at,
-                    'MMM dd, yyyy hh:mm a'
-                  )
-                }}
-              </td>
-              <td class="w-[8.75rem]">
-                {{ auditLogItem.remote_address }}
-              </td>
-            </tr>
-          </tbody>
-        </table>
-      </div>
+          <tr v-for="auditLogItem in records" :key="auditLogItem.id">
+            <td class="py-4 pr-4 break-all whitespace-nowrap">
+              {{ generateLogText(auditLogItem) }}
+            </td>
+            <td class="py-4 pr-4 break-all whitespace-nowrap">
+              {{
+                messageTimestamp(
+                  auditLogItem.created_at,
+                  'MMM dd, yyyy hh:mm a'
+                )
+              }}
+            </td>
+            <td class="py-4 w-[8.75rem]">
+              {{ auditLogItem.remote_address }}
+            </td>
+          </tr>
+        </tbody>
+      </table>
+      <TableFooter
+        :current-page="Number(meta.currentPage)"
+        :total-count="meta.totalEntries"
+        :page-size="meta.perPage"
+        class="border-slate-50 dark:border-slate-800 border-t !px-0 py-4"
+        @pageChange="onPageChange"
+      />
```

Comment:
Can we hide the table pagination while the audit log is fetching?

Reply:
Done.

Reply:
Great catch!